alf/networks/actor_networks.py

```diff
@@ -133,13 +137,31 @@ def forward(self, observation, state=()):
         """
 
         observation, state = super().forward(observation, state)
+        self._encoding_net.eval = self.eval
         encoded_obs, _ = self._encoding_net(observation)
 
         actions = []
+        i = 0
         for layer, spec in zip(self._action_layers, self._flat_action_spec):
-            action = layer(encoded_obs)
+            _info = {}
+            action = layer(encoded_obs, info=_info)
+
+            name = ('summarize_output/' + self.name + '.action_layer.' + str(i)
+                    + '.pre_activation.output_norm')
+            if self.eval:
+                name += ".eval"
+            alf.summary.scalar(name=name, data=_info["pre_activation"].norm())
+
             action = spec_utils.scale_to_spec(action, spec)
+
+            a_name = ('summarize_output/' + self.name + '.action_layer.' +
+                      str(i) + '.action.output_norm')
+            if self.eval:
+                a_name += ".eval"
+            alf.summary.scalar(name=a_name, data=action.norm())
+
```

Comment:
By default, this should not be enabled  because it might slow down training a lot.